docs/getting-started.md

```diff
@@ -97,75 +85,98 @@ components that Molecule provides. These are:
   instance.
 
-## Run test sequence commands
-
-Let's create the first Molecule managed instance with the Docker
-driver.
+The following guide will step through an example of developing and
+testing a new Ansible collection. After reading this guide, you should be
+familiar with the basics of how to use Molecule and what it can offer.
 
-First, ensure that [Docker](https://docs.docker.com/) is running with
-the typical sanity check:
+## Create a collection
 
-```bash
-$ docker run hello-world
-```
+One of the recommended ways to create a collection is to place it
+under the `collections/ansible_collections` directory.
 
-Now, we can tell Molecule to create an instance with:
+To create a collection, use the `ansible-galaxy` tool.
+`ansible-galaxy collection init <namespace>.<collection_name>`
+Example: `ansible-galaxy collection init foo.bar`
 
-```bash
-$ molecule create
-```
+After a collection is created, it would be under
+`collections/ansible_collections/<namespace>/<collection_name>`
+See [Collection structure](https://docs.ansible.com/ansible/5/dev_guide/developing_collections_structure.html#collection-structure) for details on the collection directory structure.
 
-We can verify that Molecule has created the instance and they're up and
-running with:
+## Create a role inside the collection
 
-```bash
-$ molecule list
-```
+To initialize a role inside a collection we can again use `ansible-galaxy` tool under the `roles` directory.
+`ansible-galaxy role init <namespace>.<collection_name>`
+Example: `ansible-galaxy collection init my_role`
 
-Now, let's add a task to our `tasks/main.yml` like so:
+Once the role is created, add a task under `my_role/tasks/main.yml`.
 
-```yaml
-- name: Molecule Hello World!
+```
+---
+- name: Task is running from within the role
   ansible.builtin.debug:
-    msg: Hello, World!
+    msg: "This is a task from my_role."
 ```
 
-We can then tell Molecule to test our role against our instance with:
+## Create a playbook inside the collection
 
-```bash
-$ molecule converge
-```
+Under the collection, we can create a directory named `playbooks`
+and create a new playbook file under it.
+Example: `my_playbook.yml`
 
-If we want to manually inspect the instance afterwards, we can run:
+```
+---
+- name: Test new role from within this playbook
+  hosts: localhost
+  gather_facts: false
+  tasks:
+    - name: Testing role
+      ansible.builtin.include_role:
+        name: foo.bar.my_role
+        tasks_from: main.yml
 
-```bash
-$ molecule login
 ```
 
-We now have a free hand to experiment with the instance state.
+## Adding Molecule to the Collection
 
-Finally, we can exit the instance and destroy it with:
+Inside the collection, create a directory named `extensions`
+and initialize `molecule scenario` under this directory using
+the command: `molecule init scenario <scenario_name>`
+`scenario_name` is optional, if not provided, will create
+a `default` scenario.
```

Comment:
I'd keep this as a section like it is, but convert the steps here to.. steps.  Something like:
1. Create a new directory in your collection called `extensions`.
2. Initialize a new default molecule scenario in this directory.
```
    `molecule init scenario`
```